mne/simulation/tests/test_evoked.py

```diff
@@ -52,24 +49,14 @@ def test_simulate_evoked():
     n_samples = 600
     times = np.linspace(tmin, tmin + n_samples * tstep, n_samples)
 
-    # Generate times series from 2 Morlet wavelets
-    stc_data = np.zeros((len(labels), len(times)))
-    Ws = morlet(sfreq, [3, 10], n_cycles=[1, 1.5])
-    stc_data[0][:len(Ws[0])] = np.real(Ws[0])
-    stc_data[1][:len(Ws[1])] = np.real(Ws[1])
-    stc_data *= 100 * 1e-9  # use nAm as unit
-
-    # time translation
-    stc_data[1] = np.roll(stc_data[1], 80)
-    stc = generate_sparse_stc(fwd['src'], labels, stc_data, tmin, tstep,
-                              random_state=0)
+    # Generate times series for 2 dipoles
+    stc = simulate_sparse_stc(fwd['src'], n_dipoles=2, times=times)
+    stc._data *= 1e-9
 
```

Comment:
This should be removed as the *data_fun* in *simulate_sparse_stc* is already scaling the data